projects/kit/components/input-date/multi-date/input-date-multi.directive.ts

```diff
@@ -0,0 +1,42 @@
+import {Directive} from '@angular/core';
+import {TuiDay} from '@taiga-ui/cdk';
+import {AbstractTuiTextfieldHost, tuiAsTextfieldHost} from '@taiga-ui/core';
+
+import {tuiImmutableUpdateInputDateMulti} from './immutable-update';
+import {TuiInputDateMultiComponent} from './input-date-multi.component';
+
+@Directive({
+    selector: 'tui-input-date[multiple]',
+    providers: [tuiAsTextfieldHost(TuiInputDateMultiDirective)],
+})
+export class TuiInputDateMultiDirective extends AbstractTuiTextfieldHost<TuiInputDateMultiComponent> {
+    override get value(): string {
+        return String(this.host.value);
+    }
+
+    get max(): TuiDay {
+        return this.host.computedMax;
+    }
+
+    get min(): TuiDay {
+        return this.host.computedMin;
+    }
+
+    onValueChange(value: string): void {
+        const {year, month, day} = TuiDay.parseRawDateString(value);
```

Comment:
Let's file a separate issue for i18n of this control so we don't forget about it.

Reply:
I added `TODO`